src/controllers/activegate/capability/statsd-ingest/consts.go

```diff
@@ -0,0 +1,11 @@
+package statsdingest
+
+import "github.com/Dynatrace/dynatrace-operator/src/controllers/activegate/internal/consts"
+
+const (
```

Comment:
for the rest of the codebase we have the constants in a `config.go` per package

Reply:
Makes sense. I rearranged StatsD- and other ActiveGate-related constants as suggested.